huaweicloud/services/vpc/data_source_huaweicloud_networking_port_v2.go

```diff
@@ -147,20 +142,32 @@ func dataSourceNetworkingPortV2Read(d *schema.ResourceData, meta interface{}) er
 		listOpts.DeviceID = v.(string)
 	}
 
+	return listOpts
+}
+
+func dataSourceNetworkingPortV2Read(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	config := meta.(*config.Config)
+	networkingClient, err := config.NetworkingV2Client(config.GetRegion(d))
+	if err != nil {
+		return diag.Errorf("Error creating HuaweiCloud networking client: %s", err)
```

Comment:
if we use `fmt`, `log`, `diag` package, please make sure the message does not include **[Hh]huaweicloud**